docs/webpack.dev.config.js

```diff
@@ -14,113 +14,116 @@ const appPaths = [
   path.resolve(__dirname, 'wip_modules')
 ];
 
-const config = decorateClientConfig({
-  entry: [
-    'core-js/modules/es6.symbol',
-    'react-hot-loader/patch',
-    'webpack-dev-server/client?http://localhost:3000',
-    'webpack/hot/only-dev-server',
-    path.resolve(__dirname, 'src/client-render')
-  ],
+const createConfigWithPort = (port) => {
```

Comment:
No actual changes to the config, other than wrapping it in a function that receives a port, then passes it to line 22:

`webpack-dev-server/client?http://localhost:${port}`